Formula/genext2fs.rb

```diff
@@ -0,0 +1,19 @@
+class Genext2fs < Formula
+  desc "Generates an ext2 filesystem as a normal (non-root) user"
+  homepage "http://freecode.com/projects/genext2fs"
+  url "https://downloads.sourceforge.net/projects/genext2fs/files/genext2fs/1.4.1/genext2fs-1.4.1.tar.gz"
+  sha256 "404dbbfa7a86a6c3de8225c8da254d026b17fd288e05cec4df2cc7e1f4feecfc"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
```

Comment:
Don't need this empty line. The spacing is far enough apart that leaving no empty line doesn't impact readability here.